File: code_challenge_solution/javascript/21 Calculate age in seconds.js
var ageInSeconds = function(age) {
    var secondsInMinute = 60;
    var minutesInHour = 60;
    var hoursInDay = 24;
    var daysInYear = 365;

    var seconds = age * secondsInMinute * minutesInHour * hoursInDay * daysInYear;
    return seconds;
};

console.log(ageInSeconds(30));

File: code_challenge_solution/python/21 Calculate age in seconds.py
import datetime

def calculate_age_in_seconds():
    birthdate_str = input("Enter your birthdate (YYYY-MM-DD): ")
    birthdate = datetime.datetime.strptime(birthdate_str, "%Y-%m-%d")
    current_date = datetime.datetime.now()
    age_in_seconds = (current_date - birthdate).total_seconds()
    print("Your age in seconds is:", age_in_seconds)

calculate_age_in_seconds()